src/registrar/templates/includes/form_messages.html

```diff
@@ -1,6 +1,6 @@
 {% if messages %}
 {% for message in messages %}
-<div class="usa-alert usa-alert--{{ message.tags }} usa-alert--slim margin-bottom-2">
+<div class="usa-alert usa-alert--{{ message.tags }} usa-aler margin-bottom-2">
```

Comment:
**Question:** Shouldn't this be `usa-alert`?

Reply:
It should stay usa-alert--slim. Good catch, thank you!